bin/node/testing/src/bench.rs

```diff
@@ -387,7 +387,8 @@ impl BenchDb {
 		};
 		let task_executor = TaskExecutor::new();
 
-		let backend = sc_service::new_db_backend(db_config).expect("Should not fail");
+		let backend =
+			sc_service::new_db_backend(db_config, Default::default()).expect("Should not fail");
```

Comment:
In most bench and test, I did use `default` which is the inner hashed version of trie.